pubspec.yaml

```diff
@@ -7,14 +7,14 @@ name: stack_trace
 #
 # When the major version is upgraded, you *must* update that version constraint
 # in pub to stay in sync with this.
-version: 1.9.3
+version: 1.9.4
 
 description: A package for manipulating stack traces and printing them readably.
 author: 'Dart Team <misc@dartlang.org>'
 homepage: https://github.com/dart-lang/stack_trace
 
 environment:
-  sdk: '>=1.23.0 <3.0.0'
+  sdk: '>=2.8.0-dev <3.0.0'
```

Comment:
It's totally valid to have a lower-bound of both the SDK here and in .travis.yaml of the SDK that ships the feature you're using!